crl-janitor/variables.tf

```diff
@@ -34,25 +34,15 @@ locals {
   service = "crl-janitor"
 }
 
-
 #
 # Service Account Vars
 #
 locals {
   sa_roles = [
-    # Roles needed as a part of CRL Janitor functionality.
     "roles/cloudsql.client",
-    "roles/cloudtrace.agent",
-    "roles/logging.logWriter",
-    "roles/monitoring.metricWriter",
-    # Roles used in integration testing.
-    "roles/resourcemanager.admin",
```

Comment:
I think the issue here might have been that resourcemanager.admin isn't a role.
[resourcemanager iam roles](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles)

but if you want to add them later that works for me

Reply:
yeah, I think so. For now I will remove all unneeded roles and may add them back later when needed